platform-sdk/platform-apps/tests/PlatformTestingTool/src/main/java/module-info.java

```diff
@@ -30,24 +23,27 @@
     exports com.swirlds.demo.virtualmerkle.config to
             com.fasterxml.jackson.databind;
 
-    requires static com.github.spotbugs.annotations;
-    requires java.logging;
-    requires com.swirlds.platform;
-    requires com.swirlds.fcqueue;
-    requires org.bouncycastle.provider;
-    requires com.google.protobuf;
+    requires com.fasterxml.jackson.annotation;
     requires com.fasterxml.jackson.core;
     requires com.fasterxml.jackson.databind;
-    requires com.fasterxml.jackson.annotation;
-    requires org.apache.logging.log4j;
-    requires org.apache.logging.log4j.core;
-    requires lazysodium.java;
+    requires com.google.protobuf;
+    requires com.swirlds.common.test.fixtures;
+    requires com.swirlds.common.testing;
+    requires com.swirlds.common;
+    requires com.swirlds.fchashmap;
+    requires com.swirlds.fcqueue;
+    requires com.swirlds.jasperdb;
+    requires com.swirlds.logging;
+    requires com.swirlds.merkle.test;
+    requires com.swirlds.merkle;
+    requires com.swirlds.platform.core;
+    requires com.swirlds.platform.gui;
+    requires com.swirlds.virtualmap;
+    requires java.logging;
     requires java.management;
+    requires java.sql;
```

Comment:
Where is this used???

Reply:
I do not know. I did not add it, I just sorted the entries alphabetically because they were not ordered in some useful way.

Because it is a module from the SDK, the Gradle dependency analysis does not help here. I think this can probably be removed. Should I remove it, or can you take a not so that someone with more context can check this? There are also more "requires java.sql;" in the 'platform-sdk' code base. Maybe some of these are also redundant?

Reply:
lol, I created https://github.com/hashgraph/hedera-services/issues/8071